agent/engine/engine_unix_integ_test.go

```diff
@@ -1526,3 +1528,69 @@ func TestFluentdTag(t *testing.T) {
 	err = utils.SearchStrInDir(logdir, "ecsfts", logTag)
 	require.NoError(t, err, "failed to find the log tag specified in the task definition")
 }
+
+func TestDockerExecAPI(t *testing.T) {
+	testTimeout := 1 * time.Minute
+	taskEngine, done, _ := setupWithDefaultConfig(t)
+	defer done()
+
+	stateChangeEvents := taskEngine.StateChangeEvents()
+
+	taskArn := "testDockerExec"
+	testTask := createTestTask(taskArn)
+
+	A := createTestContainerWithImageAndName(baseImageForOS, "A")
+
+	A.EntryPoint = &entryPointForOS
+	A.Command = []string{"sleep 30"}
+	A.Essential = true
+
+	testTask.Containers = []*apicontainer.Container{
+		A,
+	}
+	execConfig := types.ExecConfig{
+		AttachStdin:  false,
+		AttachStderr: false,
+		AttachStdout: false,
```

Comment:
nit: you can omit all these values as false is the default (same for `DetachKeys` and `Env`)

Reply:
Fixed it